packages/trader/src/Stores/base-store.ts

```diff
@@ -1,7 +1,18 @@
 import { action, intercept, observable, reaction, toJS, when, makeObservable } from 'mobx';
 import { isProduction, isEmptyObject } from '@deriv/shared';
-
+import { TCoreStores } from '@deriv/stores/types';
 import Validator from 'Utils/Validator';
+import { getValidationRules } from './Modules/Trading/Constants/validation-rules';
+
+type TValidationRules = ReturnType<typeof getValidationRules> | Record<string, never>;
```

Comment:
what's the point of `Record<string, never>` ?

Reply:
@henry-deriv  it's representing an empty object which is validation_rules's initial value in this store :)